code/game/objects/items/motion_detector.dm

```diff
@@ -1,353 +1,149 @@
-
-#define MOTION_DETECTOR_LONG 0
-#define MOTION_DETECTOR_SHORT 1
-
-#define MOTION_DETECTOR_HOSTILE ""
-#define MOTION_DETECTOR_FRIENDLY "_friendly"
-#define MOTION_DETECTOR_DEAD "_dead"
-#define MOTION_DETECTOR_FUBAR "_fubar" //i.e. can't be revived. Might have useful gear to loot though!
-
-
-/obj/effect/detector_blip
+/obj/effect/edge_blip
 	icon = 'icons/Marine/marine-items.dmi'
-	icon_state = "detector_blip"
-	layer = BELOW_FULLSCREEN_LAYER
+	plane = ABOVE_HUD_PLANE
```

Comment:
above hud ?

Reply:
yeah, hence it's hidden by the darkness plane. And yeah, above hud, it's often on the edges of the screen where it could be hidden by the hud too